changelog.md

```diff
@@ -5,6 +5,12 @@ All notable changes done in ODS library will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Changed
+
+- \[Lib\] Use `OptIn` annotation instead of propagating `ExperimentalMaterialApi` and `ExperimentalPagerApi` ([#320](https://github.com/Orange-OpenSource/ods-android/issues/320))
```

Comment:
Is it something to display in the changelog cause this has no impact for the user?

Reply:
A user of the library would probably have added some `@ ExperimentalMaterialApi ` or `@ExperimentalPagerApi` annotation to use our APIs. Now she/he would be able to remove these annotations. But I admit this is a very light impact.